gui/src/app/facilities/facilities-detail/facilities-detail.component.html

```diff
@@ -44,20 +44,20 @@ <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
         <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
         <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="hover-dark"></tr>
       </table>
-
-      <button class="ml-2" mat-raised-button color="accent" (click)="deleteFacility()">
-        {{'FACILITIES.DELETE_FACILITY' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="editFacility()">
-        {{'FACILITIES.EDIT_FACILITY' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="addFacilityAdmin()">
-        {{'FACILITIES.ADD_ADMIN' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="editAdmins()">
-        {{'FACILITIES.EDIT_ADMIN' | translate}}
-      </button>
-
+      <div class="mt-4">
+        <button class="ml-2" mat-raised-button color="accent" (click)="deleteFacility()">
+          {{'FACILITIES.DELETE_FACILITY' | translate}}
+        </button>
+        <button class="ml-2" mat-raised-button color="accent" (click)="editFacility()">
```

Comment:
Please, change the colors of these buttons. Accent should be used only for those buttons that are probable to be used by a user. Make those buttons white and also, move them to the right side of the page. Take a look at the request-detail page.